modules/management-console-agent/src/main/java/org/apache/ignite/agent/dto/action/query/RunningQuery.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.agent.dto.action.query;
+
+import org.apache.ignite.internal.processors.cache.query.GridCacheQueryType;
+import org.apache.ignite.internal.util.typedef.internal.S;
+
+/**
+ * Descriptor of running query.
+ */
+public class RunningQuery {
+    /** */
+    private long id;
+
+    /** Query text. */
+    private String qry;
+
+    /** Query type. */
+    private GridCacheQueryType qryType;
+
+    /** Schema name. */
+    private String schemaName;
+
+    /** */
+    private long startTime;
+
+    /** */
+    private long duration;
+
```

Comment:
Please mention what timeunit is supposed to be used here?
Is this in milliseconds? In seconds? Smth else?

Reply:
And it can be even better to specify timeunits in Getter and Setter methods